libraries/AP_BattMonitor/AP_BattMonitor_Analog.cpp

```diff
@@ -48,6 +48,13 @@ const AP_Param::GroupInfo AP_BattMonitor_Analog::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("AMP_OFFSET", 5, AP_BattMonitor_Analog, _curr_amp_offset, AP_BATT_CURR_AMP_OFFSET_DEFAULT),
 
+    // @Param: AMP_AUTOCAL
+    // @DisplayName: automatic AMP offset calibration
+    // @Description: When enabled this will set BATTn_AMP_OFFSET to the current voltage reading on the CURR_PIN at boot. This is useful for some types of hall effect sensors.
```

Comment:
```suggestion
    // @Description: When enabled this will set BATTn_AMP_OFFSET to the current voltage reading on the CURR_PIN at boot. This is useful for some types of high current hall effect sensors in which the much smaller current drain of ancillary peripherals can be ignored.
```